### PR TITLE
M2-5201: Update readme file

### PR DESCRIPTION
**Pull request by ChaconC**
## Description

[JIRA: M2-5201](https://mindlogger.atlassian.net/browse/M2-5201)

Update README and add LICENSE.md

Pending LICENSE.md review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,38 @@
-# Mind Logger Admin Redesigned
+# ChildMindInstitute - MindLogger Admin
```
Comment (farmerpaul) [inline README.md]:
```suggestion
# Child Mind Institute - MindLogger Admin
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This repository is used for the Admin Panel of the MindLogger application stack
```
Comment (farmerpaul) [inline README.md]:
```suggestion
This repository is used for the Admin Panel of the [MindLogger](https://mindlogger.org/) application stack.
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+## Getting Started
+
+* MindLogger Admin - **This Repo**
+* MindLogger Backend - [GitHub Repo](https://github.com/ChildMindInstitute/mindlogger-backend-refactor)
+* MindLogger Mobile App - [GitHub Repo](https://github.com/ChildMindInstitute/mindlogger-app-refactor)
+* MindLogger Web App - [GitHub Repo](https://github.com/ChildMindInstitute/mindlogger-web-refactor)
+
+## Getting Started
+
+Running the app
```
Comment (farmerpaul) [inline README.md]:
```suggestion
Running the app:
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,38 @@
...
+
+### 1. Prerequisites
+
+- NodeJS `18` or higher, recommend using `asdf` or `nvm` to manage local node version
+- [Backend](https://github.com/ChildMindInstitute/mindlogger-backend-refactor) project running locally or accessible in a test environment
+- Configured [environment variables](#environment-variables)\
```
Comment (farmerpaul) [inline README.md]:
```suggestion
- Configured [environment variables](#environment-variables):\
```

File: README.md [hunk 5]
```diff
@@ -1,8 +1,38 @@
...
+* MindLogger Web App - [GitHub Repo](https://github.com/ChildMindInstitute/mindlogger-web-refactor)
+
+## Getting Started
+
+Running the app
+
+### 1. Prerequisites
+
+- NodeJS `18` or higher, recommend using `asdf` or `nvm` to manage local node version
+- [Backend](https://github.com/ChildMindInstitute/mindlogger-backend-refactor) project running locally or accessible in a test environment
```
Comment (farmerpaul) [inline README.md:19]:
```suggestion
- [Backend](https://github.com/ChildMindInstitute/mindlogger-backend-refactor) project running locally or accessible in a test environment
  - If running locally, ensure that `http://localhost:3000` has been added to the BE's `CORS__ALLOW_ORIGINS` environment variable
```

File: README.md [hunk 6]
```diff
@@ -1,8 +1,38 @@
...
+- Configured [environment variables](#environment-variables)\
+  `cp .env.example .env`
+
+### 2. Run the app
+
+- Install dependencies using `npm install`
+- Run the project `npm run start`. See [scripts](#available-scripts)
```
Comment (farmerpaul) [inline README.md]:
```suggestion
- Run the project using `npm run start`. See [scripts](#available-scripts)
```

File: README.md [hunk 7]
```diff
@@ -1,8 +1,38 @@
...
+- Launch browser [http://localhost:3000](http://localhost:3000) to view the admin panel
+
+## Features
+
+What are MindLogger's current capabilities? [Knowledge Base Article](https://mindlogger.atlassian.net/servicedesk/customer/portal/3/topic/4d9a9ad4-c663-443b-b7fc-be9faf5d9383/article/337444910)
```
Comment (farmerpaul) [inline README.md]:
```suggestion
See MindLogger's [Knowledge Base article](https://mindlogger.atlassian.net/servicedesk/customer/portal/3/topic/4d9a9ad4-c663-443b-b7fc-be9faf5d9383/article/337444910) to discover the MindLogger application stack's features.
```

File: README.md [hunk 8]
```diff
@@ -15,35 +45,57 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
```
Comment (farmerpaul) [inline README.md:46]: I'd suggest a better description of `React-app-rewired` above: `All the benefits of create-react-app without the limitations of "no config"`

File: README.md [hunk 9]
```diff
@@ -15,35 +45,57 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
+    Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+
+    You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+
+### Testing
+
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- `npm run test` - Launches the test runner in the interactive watch mode.\
```
Comment (farmerpaul) [inline README.md]:
```suggestion
- `npm run test` - Launches the test runner in the interactive watch mode.
```

File: README.md [hunk 10]
```diff
@@ -15,35 +45,57 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
+- `npm run test:nowatch` - Runs test suite once with the `watchAll=false` flag
 
-### `npm run eject`
+## Environment Variables
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+| Key                                | Required      | Default value      | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| REACT_APP_API_DOMAIN               | yes | null | MindLogger Backend API endpoint |
+| REACT_APP_ENV                      | no | null | Check if the app is in `prod` or `staging`  |
```
Comment (farmerpaul) [inline README.md]:
```suggestion
| REACT_APP_ENV                      | no | null | Environment to run the app in (`prod` or `staging`)  |
```

File: README.md [hunk 11]
```diff
@@ -15,35 +45,57 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
+- `npm run test:related` - Runs tests only for modified files
+- `npm run test:nowatch` - Runs test suite once with the `watchAll=false` flag
 
-### `npm run eject`
+## Environment Variables
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+| Key                                | Required      | Default value      | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| REACT_APP_API_DOMAIN               | yes | null | MindLogger Backend API endpoint |
```
Comment (farmerpaul) [inline README.md]:
```suggestion
| REACT_APP_API_DOMAIN               | yes | null | MindLogger Backend API base URL |
```

File: .node-version [hunk 12]
```diff
@@ -0,0 +1 @@
+v20.11.1
```
Comment (farmerpaul) [inline .node-version]: The README says to use Node `18` or higher – does it make sense to require version `20` here (and in `.nvmrc`)?

**Review by farmerpaul — APPROVED** [review]
Looks great @ChaconC! I just identified a few nits/suggestions. Mostly unimportant things like punctuation or wording adjustments. Happy to preapprove, but would be best to get another set of eyes from ScienceSoft before merging!

File: .node-version (hunk 12 above)
Comment (ChaconC) [reply .node-version]: I put `20` as it's the current LTS version, but personally had issues running with `16` that went away once I moved to 18 or 20...
So this is more of a suggested (`20`) vs required (`18`) 🤔 

Edit: Will go with `18` as that's already in the repo

Comment (farmerpaul) [reply .node-version]: Ah okay sure, fine with me 🙂

File: README.md [hunk 13]
```diff
@@ -1,8 +1,38 @@
-# Mind Logger Admin Redesigned
+# ChildMindInstitute - MindLogger Admin
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This repository is used for the Admin Panel of the MindLogger application stack
+
+## Getting Started
```
Comment (farmerpaul) [inline README.md:5]: Just notice you have another `## Getting Started` heading below. Maybe this should be:
```suggestion
## Application Stack
```
?

File: .node-version (hunk 12 above)
Comment (sultanofcardio) [inline .node-version]: I think they're using node v18, and I added this in #1472 so we could probably remove this file

File: .nvmrc [hunk 14]
```diff
@@ -0,0 +1 @@
+v20.11.1
```
Comment (sultanofcardio) [inline .nvmrc]: To match the `.node-version` file

```suggestion
v18.19.0
```

https://github.com/ChildMindInstitute/mindlogger-admin/blob/57051d6026285cb4433900db512861886524f04e/.node-version#L1

File: README.md (hunk 13 above)
Comment (ChaconC) [reply README.md:5]: Nice catch! problem with copying sections

**Pull request merged**